.dockerignore

```diff
@@ -0,0 +1,4 @@
+.git
+*Dockerfile*
+*docker-compose*
+node_modules
```

Comment:
Just tested it out, needs `*/node_modules` to be properly ignored.

Reply:
ecf6404